tests/KafkaFlow.IntegrationTests/Core/Handlers/MessageStorage.cs

```diff
@@ -139,6 +147,51 @@ public static async Task AssertNullMessageAsync()
             await Task.Delay(100).ConfigureAwait(false);
         }
     }
+    
+    public static async Task AssertMessageAsync(OffsetTrackerMessage message)
+    {
+        var start = DateTime.Now;
+
+        while (!s_offsetTrackerMessages.Any(x => x.Id == message.Id && x.Offset == message.Offset))
+        {
+            if (DateTime.Now.Subtract(start).Seconds > TimeoutSec)
+            {
+                Assert.Fail("Message (OffsetTrackerMessage) not received");
+                return;
+            }
+
+            await Task.Delay(100).ConfigureAwait(false);
+        }
+    }
+    
+    public static async Task AssertOffsetTrackerMessageAsync(OffsetTrackerMessage message, bool assertInStore = true)
```

Comment:
The method could be renamed to something like `AssertOffsetTrackerMessageNotReceivedAsync` so its more clear the method purpose.

Reply:
Renamed to `AssertOffsetTrackerMessageNotReceivedAsync`